dracut/modules.d/99kiwi-lib/kiwi-luks-lib.sh

```diff
@@ -16,6 +16,12 @@
     return 1
 }
 
+# Return true if there is a generated systemd service unit for our LUKS volume
+function luks_has_generated_systemd_unit {
+    eval $(systemctl show -p UnitFileState,SubState systemd-cryptsetup@luks)
+    test "$UnitFileState" = "generated" -a "$SubState" = "exited"
```

Comment:
Okay, this is a potential issue in case systemctl show fails for some reason.

Maybe let's put these lines aboce the systemctl line:

    local UnitFileState=
    local SubState=
    eval $(systemctl show  .... )